src/metric.jl

```diff
@@ -1,6 +1,6 @@
 abstract type AbstractMetric end
 
-_string_M⁻¹(mat::AbstractMatrix, n_chars::Int=32) = _string_M⁻¹(LinearAlgebra.diag(mat), n_chars)
+_string_M⁻¹(mat::AbstractMatrix, n_chars::Int=32) = _string_M⁻¹(diag(mat), n_chars)
```

Comment:
Minor comment: Any reason for reversing this change made in a previous PR?

Reply:
This one is not reverting. We previsouly decided to expliclty using functions from `LinearAlgebra`, e.g. `diag` so the variable `LinearAlgebra` is not defined now. This is a bug which was not catched. I'm happy to move it to seperate PR if you think this make the history clear.